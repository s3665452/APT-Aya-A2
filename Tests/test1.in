UYLLLULRRUUBUULRYUUYLBLRRLBRYLLURRRYYYLYBYULYLRYYYBUUBYRRLUYURBRRBURBBRLLUYLBLURLBBBUYYRRBBBBUYYBULB
Player1
Player2
turn 1 L 3
turn 5 R 4
turn 3 B 1
turn 0 L 5
turn 4 U 5
turn 2 L 2
turn 0 U 4
