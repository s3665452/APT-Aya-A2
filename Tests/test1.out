Factories:
0: F U L R
1: 
2: R U L L 
3: U B R Y 
4: U U U U U
5: 

Score for Player1: 0
Mosaic for A:
1:         . || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken:

Score for Player2: 0
Mosaic for B:
1:         . || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   R R R R || . . . . .
5: . . . . . || . . . . .
broken: R

Factories:
0: 
1: 
2: R U L L 
3: 
4: U U U U U 
5: 

Score for Player1: 0
Mosaic for A:
1:         B || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   . . . . || . . . . .
5: . . . . . || . . . . .
broken:

Score for Player2: 0
Mosaic for B:
1:         . || . . . . .
2:       . . || . . . . .
3:     . . . || . . . . .
4:   R R R R || . . . . .
5: L L L L L || . . . . .
broken: F

Factories:
0: 
1: 
2:  
3: 
4: 
5: 

Score for Player1: 0
Mosaic for A:
1:         B || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   . . . . || . . . . .
5: U U U U U || . . . . .
broken:

Score for Player2: 0
Mosaic for B:
1:         . || . . . . .
2:       L L || . . . . .
3:     . . . || . . . . .
4:   R R R R || . . . . .
5: L L L L L || . . . . .
broken: F

Factories:
0: L L R 
1: R Y Y R 
2: R U Y B 
3: R R L B 
4:  
5: U Y L R 

Score for Player1: 0
Mosaic for A:
1:         B || . . . . .
2:       . . || . . . . .
3:     L L L || . . . . .
4:   U U U U || . . . . .
5: U U U U U || . . . . .
broken:
